pallets/parachain-staking/src/set.rs

```diff
@@ -160,45 +184,50 @@ impl<T: Ord> OrderedSet<T> {
 	pub fn into_vec(self) -> Vec<T> {
 		self.0
 	}
+
+	/// Sort from greatest to lowest
+	pub fn sort_greatest_to_lowest(&mut self) {
+		self.0.sort_by(|a, b| b.cmp(a));
+	}
 }
 
-impl<T: Ord> From<Vec<T>> for OrderedSet<T> {
+impl<T: Ord + Debug> From<Vec<T>> for OrderedSet<T> {
```

Comment:
why do we need Debug here?

Reply:
It can be removed again, I needed it for debugging bugs.

Reply:
Fixed in c11ad27